src/main/java/util/WriterReader.java

```diff
@@ -0,0 +1,71 @@
+package util;
+
+import exceptions.AutomationException;
+
+import java.io.*;
+
+public class WriterReader {
+
+
+    private static final String FILE_NAME = "fileUtils.txt";
+
+    public static void saveUtils() {
+
+
+        FileUtils p1 = new FileUtils(WebDriverBrowsertack.getToken());
+
+        try {
+            FileOutputStream f = new FileOutputStream(new File(FILE_NAME));
+            ObjectOutputStream o = new ObjectOutputStream(f);
+
+            o.writeObject(p1);
+
+            o.close();
```

Comment:
put the closing stream sequence in a finally block.

Reply:
moving object creation in twr block, and deleting close calls